exporter/datadogexporter/internal/metrics/system.go

```diff
@@ -0,0 +1,134 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package metrics // import "github.com/open-telemetry/opentelemetry-collector-contrib/exporter/datadogexporter/internal/metrics"
+
+import (
+	"strings"
+
+	"gopkg.in/zorkian/go-datadog-api.v2"
+)
+
+// copyMetric copies the metric from src by giving it a new name. If div differs from 1, it scales all
+// data points. If interval is non-zero, it sets it as the metric's interval.
+//
+// Warning: this is not a deep copy. Only some fields are fully copied, others remain shared. This is intentional.
+// Do not alter the returned metric (or the source one) after copying.
+func copyMetric(src datadog.Metric, name string, div float64, interval int) datadog.Metric {
+	cp := src
+	cp.Metric = &name
+	if interval != 0 {
+		cp.Interval = &interval
+	}
+	t := "gauge"
+	cp.Type = &t
+	if div == 0 || div == 1 || len(src.Points) == 0 {
+		// division by 0 or 1 should not have an impact
+		return cp
+	}
+	cp.Points = make([]datadog.DataPoint, len(src.Points))
+	for i, dp := range src.Points {
+		cp.Points[i][0] = dp[0]
+		if dp[1] != nil {
+			newdp := *dp[1] / div
+			cp.Points[i][1] = &newdp
+		}
+	}
+	return cp
+}
+
+// mebibytes is the number of bytes in a mebibyte
+const mebibytes = 1024 * 1024
+
+// extractSystemMetrics takes an OpenTelemetry metric m and extracts Datadog system metrics from it,
+// if m is a valid system metric. The boolean argument reports whether any system metrics were extractd.
+func extractSystemMetrics(m datadog.Metric) (ms []datadog.Metric, ok bool) {
```

Comment:
I don't understand how does this fix the issue with that specific metric, could you elaborate on that? It would also be nice to have this on the PR description, so that we are able to understand issues on end-users using v0.61.0

Reply:
I have updated the PR description by introducing a hyperlink to make it more obvious what I meant, and added another sentence with another problem this is fixing. I hope that is more clear.

I would not be able to tell you _exactly_ why the metrics transform processor was causing the issue in Kubernetes but I suspect it was that [hacky config](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/datadogexporter/internal/metrics/transform.go#L148-L180). 

Let me know if this is still unclear.